js/utils/metrics.js

```diff
@@ -0,0 +1,111 @@
+import MetricsModal from '../views/modals/MetricsModal';
+import app from '../app';
+import { version } from '../../package.json';
+import * as os from 'os';
+import { getCurrentConnection } from './serverConnect';
+import { remote } from 'electron';
+
+
+let metricsRestartNeeded = false;
+
+export function isMetricRestartNeeded() {
+  return metricsRestartNeeded;
+}
+
+export function addMetrics() {
+  function loadMetrics() {
+    // Reverse the countly opt out
+    window.localStorage.setItem('cly_ignore', 'false');
```

Comment:
What is this purpose of this? It doesn't seem to be used anywhere. Is this a flag that countly is reading?

Reply:
That's a countly flag. Their code assumes if you opt out you can't change your mind, we have to change the flag to re-activate the tracking if the user wants to turn it back on.